assets/chat/js/chat.js

```diff
@@ -658,13 +658,13 @@ class Chat {
         return this;
     }
 
-    withViewerStates(viewerStates) {
-        viewerStates.forEach(state => this.onVIEWERSTATE(state));
+    async withViewerStates(viewerStates) {
+        if(viewerStates) viewerStates.forEach(state => this.onVIEWERSTATE(state));
```

Comment:
Dev Chat would error out with this async/await code, this check is to fix that. As history loads in Dev Chat, viewer states should load when given data and not null.

With Whispers doesn't have the data about whether the user is authenticated or not because it is all loaded at the same time, but there is no delay added for users who aren't signed in because a 503 error is returned and for users signed in it loads as normal.

Connect is changed to async to allow for .then